suricata.yaml.in

```diff
@@ -137,6 +137,7 @@ outputs:
             # custom allows additional http fields to be included in eve-log
             # the example below adds three additional fields when uncommented
             #custom: [Accept-Encoding, Accept-Language, Authorization]
+            # dump-all-headers: set this value to {both, request, response} to dump all http headers for every http request and/or response
```

Comment:
formatting a is a bit odd. Think it's better to have the comment on separate lines, then the option (commented out) on a new line so it can easily be enabled by the user